Makefile

```diff
@@ -126,11 +177,11 @@ clean: $(CLEAN_TARGETS) ## Runs all clean-* targets.
 # -------------------------------------------------------------------
 LDFLAGS=-ldflags "-X ${PACKAGE_NAME}/app.Commit=${COMMIT} -X ${PACKAGE_NAME}/app.BuildTime=${BUILD_TIME}"
 
-$(SERVER_BIN): ## Build the server
+$(SERVER_BIN):prebuild-check deps generate ## Build the server
 	@echo "building $(SERVER_BIN)..."
 	$(GO_BIN) build -v $(LDFLAGS) -o $(SERVER_BIN)
 
-.PHONY: build
+.PHONY: build generate
```

Comment:
Please do not add this recipe. I don't want to always run generate when I want to build. This potentially slows down CI like hell because it calls `make generate` manually.

Reply:
@kwk  so this would make user to do `make generate` manually first before doing `make build` right ?

Reply:
yes